dist/meteo-france-weather-card.js

```diff
@@ -99,15 +99,18 @@ function getVigilance(color, alertEntity) {
 
 function getRainForecast(rainForecastEntity) {
   
-  let rainForecastColors = ["rgba(95,155,234,0.1)", "rgba(95,155,234,0.4)", "rgba(95,155,234,0.7)", "rgba(95,155,234,1)"];
-  let rainForecastTexts = ["Pas de pluie", "Pluie faible", "Pluie modérée", "Pluie forte"];
-  
+  let rainForecastColors = new Map([
+	    ['Temps sec', "0.1"],
+	    ['Pluie faible', "0.4"],
+	    ['Pluie modérée', "0.7"],
+	    ['Pluie forte',    "1"]
+  ]);
+
   let rainForecastList = [];
-  for (let [time, value] of Object.entries(rainForecastEntity.attributes)) {
-    time = time.split("_")[2]
+  for (let [time, value] of Object.entries(rainForecastEntity.attributes['1_hour_forecast'])) {
     if(time != undefined && time.match(/[0-9]*min/g)) {
-      time = time.substring(0, time.length-3);
-      rainForecastList.push([time, rainForecastColors[value-1], rainForecastTexts[value-1]]);
+      time = time.substring(0, time.length-4);
```

Comment:
4 represents " min" ?

you can simply do something like:
```suggestion
      time = time.replace('min', '').trim();
```